### PR TITLE
Tweak - Added links to shipping methods configuration in shipping zones.

### PR DESCRIPTION
**Pull request by seostudio**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes # .

### How to test the changes in this Pull Request:

1. Create shipping zone
2. Add shipping method 
3. Go to Shipping zones - see link to shipping method config on shipping method title.

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Added links on shipping method titles on shipping zones configuration page.

image

### CODE REVIEW

File: includes/admin/settings/views/html-admin-page-shipping-zones.php [hunk 1]
```diff
@@ -37,9 +37,14 @@
...
 
 					if ( ! empty( $methods ) ) {
-						foreach ( $methods as $method ) {
+						foreach ( $methods as $instance_id => $method ) {
 							$class_name = 'yes' === $method->enabled ? 'method_enabled' : 'method_disabled';
-							echo '<li class="wc-shipping-zone-method ' . esc_attr( $class_name ) . '">' . esc_html( $method->get_title() ) . '</li>';
+
+							$shipping_method_link_or_title = esc_html( $method->get_title() );
+							if ( $method->has_settings() ) {
+								$shipping_method_link_or_title = '<a href="admin.php?page=wc-settings&amp;tab=shipping&amp;instance_id=' . $instance_id . '">' . $shipping_method_link_or_title . '</a>';
```
Comment (rrennick) [inline includes/admin/settings/views/html-admin-page-shipping-zones.php:45]: @seostudio Thanks for submitting the PR. Could you use `admin_url()` to generate this link and ensure the URL is properly escaped?

Comment (seostudio) [reply includes/admin/settings/views/html-admin-page-shipping-zones.php]: Fixed.

**Comment by Konamiman** [comment]
Hi @seostudio, apologies for the huge delay in the review of your pull request. I just tested it and it indeed looks like a great improvement, thank you very much.

However we can't merge your pull request as is. Since you created it the WooCommerce code structure has changed (now all the code is inside the `plugins/woocommerce` directory, [some background information](https://developer.woocommerce.com/2021/10/14/rfc-woocommerce-admin-is-merging-into-core/)) and the files you modified aren't at their original locations anymore.

So, could you please fix the merge conflicts and ping me so that I can merge the pull request? For reference, these are the new locations for the files:

* [woocommerce/plugins/woocommerce/legacy/js/admin/wc-shipping-zones.js](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/legacy/js/admin/wc-shipping-zones.js) -  Important: note that the file is inside the `legacy` folder! The file in `assets` that you modified is no longer under source control, it's autogenerated when WooCommerce is built.

* [woocommerce/plugins/woocommerce/includes/admin/settings/views/html-admin-page-shipping-zones.php](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/admin/settings/views/html-admin-page-shipping-zones.php)

Since your changeset is small perhaps you might find it easier to close this pull request and create a new one instead. It's your choice.

**Comment by samueljseay** [comment]
We are closing this pull request, as the last activity has been quite some time ago and we have not received a response from you. If you are still interested or you feel like this is a mistake, please open another PR based on a recent version of the trunk branch! Thanks!

**Pull request closed without merging**